src/Synergism.ts

```diff
@@ -858,6 +858,9 @@ const loadSynergy = async () => {
         Object.defineProperty(window, 'Decimal', {
             value: Decimal
         });
+        if (data) {
+            data.exporttest = false;
```

Comment:
If we're attaching the variables to window, we want to mark the save as a testing save. Otherwise it inherits the value from the loaded save file.